go.mod

```
@@ -21,12 +21,12 @@ require (
 	golang.org/x/exp v0.0.0-20220722155223-a9213eeb770e
 	golang.org/x/sys v0.16.0
 	gomodules.xyz/jsonpatch/v2 v2.4.0
-	k8s.io/api v0.29.0
+	k8s.io/api v0.29.1
 	k8s.io/apiextensions-apiserver v0.29.0
```

Also bump this (and the same in the example)?
```suggestion
	k8s.io/apiextensions-apiserver v0.29.1
```

---

Yeah. I'll open a PR in a bit